samples/bluetooth/central_uart/src/main.c

```diff
@@ -384,12 +384,16 @@ static void connected(struct bt_conn *conn, uint8_t conn_err)
 		LOG_WRN("MTU exchange failed (err %d)", err);
 	}
 
+#ifdef CONFIG_BT_NUS_SECURITY_ENABLED
```

Comment:
You could use here IS_ENABLED macro instead of conditional compilation

```suggestion
if(IS_ENABLED(CONFIG_BT_NUS_SECURITY_ENABLED)) {
```